### PR TITLE
config: support online change gc worker config

### PR DESCRIPTION
**Pull request by NingLin-P**
###  What have you changed?

This pr is part of online config change.
This pr add support for online change gc worker config by using `VersionTrack` to listen the update of config, this pr also make the `Debugger` using the same way to change io limit.

###  What is the type of the changes?

- New feature (a change which adds functionality)

###  How is the PR tested?

- [ ] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

###  Does this PR affect documentation (docs) or should it be mentioned in the release notes?

No

###  Does this PR affect `tidb-ansible`?

No

### CODE REVIEW

**Comment by overvenus** [comment]
LGTM, please resolve conflicts.

File: src/server/gc_worker.rs [hunk 1]
```diff
@@ -1923,41 +1958,133 @@ mod tests {
...
+        // update of other module's config should not effect gc worker config
+        let mut incoming = cfg.clone();
+        incoming.raft_store.raft_log_gc_threshold = 2000;
+        let rollback = cfg_controller.update_or_rollback(incoming).unwrap();
+        assert!(rollback.right().unwrap());
+        validate(&scheduler, move |cfg: &GcConfig, _| {
+            assert_eq!(cfg, &GcConfig::default());
+        });
+
+        // update cg worker config
```
Comment (MyonKeminta) [inline src/server/gc_worker.rs]:
```suggestion
        // Update gc worker config
```

**Pull request merged**